Checkout/Source/Tokenisation/CheckoutAPIService.swift

```diff
@@ -170,3 +171,53 @@ final public class CheckoutAPIService: CheckoutAPIProtocol {
     }
   }
 }
+
+// MARK: Security Code Request
+
+extension CheckoutAPIService {
+  public func createSecurityCodeToken(securityCode: String, completion: @escaping (Result<SecurityCodeTokenDetails, TokenisationError.SecurityCodeError>) -> Void) {
+    guard !publicKey.isEmpty else {
+        completion(.failure(.missingAPIKey))
+        return
+    }
+
+    let request = SecurityCodeRequest(tokenData: TokenData(securityCode: securityCode))
+    let requestParameterResult = requestFactory.create(request: .securityCodeToken(request: request, publicKey: publicKey))
+
+    switch requestParameterResult {
+    case .success(let requestParameters):
+//      logManager.queue(event: .tokenRequested(CheckoutLogEvent.TokenRequestData(
+//        tokenType: request.type,
+//        publicKey: publicKey
+//      )))
+      createSecurityCodeToken(requestParameters: requestParameters, completion: completion)
+    case .failure(let error):
+      switch error {
+      case .baseURLCouldNotBeConvertedToComponents, .couldNotBuildURL:
+        completion(.failure(.couldNotBuildURLForRequest))
+      }
+    }
+  }
+
+  private func createSecurityCodeToken(requestParameters: NetworkManager.RequestParameters, completion: @escaping (Result<SecurityCodeTokenDetails, TokenisationError.SecurityCodeError>) -> Void) {
+    requestExecutor.execute(
+      requestParameters,
+      responseType: SecurityCodeResponse.self,
+      responseErrorType: TokenisationError.ServerError.self
+    ) { [tokenDetailsFactory, logManager, logTokenResponse] tokenResponseResult, httpURLResponse in
+//      logTokenResponse(tokenResponseResult, httpURLResponse)
```

Comment:
Is this intentionally commented ?

Reply:
yeah. will be done during the logging ticket.